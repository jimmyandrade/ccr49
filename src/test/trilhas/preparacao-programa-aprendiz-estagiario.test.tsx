import TraineeTrack from "../../pages/trilhas/preparacao-programa-aprendiz-estagiario";
import { create } from "react-test-renderer";

jest.mock("next/image", () => "Image");

describe("Trainee Track page", () => {
    it("should match snapshot", () => {
        const testRenderer = create(<TraineeTrack />);
        expect(testRenderer).toMatchSnapshot();
    })
})