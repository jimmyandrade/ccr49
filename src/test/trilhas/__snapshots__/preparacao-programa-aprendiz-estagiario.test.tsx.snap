// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Trainee Track page should match snapshot 1`] = `
<main
  className="MuiContainer-root MuiContainer-maxWidthLg"
>
  <h2
    className="MuiTypography-root MuiTypography-h4"
  >
    Trilhas
  </h2>
  <p
    className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom MuiTypography-paragraph"
  >
    Menu da 
    <i
      lang="en"
    >
      Trilha
    </i>
  </p>
  <div
    className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-1"
  >
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-10 MuiGrid-grid-sm-4"
    >
      <div
        className="MuiPaper-root MuiCard-root MuiPaper-elevation1 MuiPaper-rounded"
      >
        <div
          className="MuiCardContent-root"
        >
          <h3
            className="MuiTypography-root MuiTypography-h6"
          >
            Matérias da Trilha
          </h3>
        </div>
        <ul
          className="MuiList-root MuiList-padding"
        >
          <div
            aria-disabled={false}
            className="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
            onBlur={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            role="button"
            tabIndex={0}
          >
            <div
              className="MuiListItemText-root"
            >
              <span
                className="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
              >
                Inglês Técnico
              </span>
            </div>
          </div>
          <div
            aria-disabled={false}
            className="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
            onBlur={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            role="button"
            tabIndex={0}
          >
            <div
              className="MuiListItemText-root"
            >
              <span
                className="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
              >
                Gestão de Projetos
              </span>
            </div>
          </div>
          <div
            aria-disabled={false}
            className="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
            onBlur={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            role="button"
            tabIndex={0}
          >
            <div
              className="MuiListItemText-root"
            >
              <span
                className="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
              >
                Gestão de tempo
              </span>
            </div>
          </div>
          <div
            aria-disabled={false}
            className="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
            onBlur={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            role="button"
            tabIndex={0}
          >
            <div
              className="MuiListItemText-root"
            >
              <span
                className="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
              >
                Finanças corporativa e Pessoal
              </span>
            </div>
          </div>
          <div
            aria-disabled={false}
            className="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
            onBlur={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            role="button"
            tabIndex={0}
          >
            <div
              className="MuiListItemText-root"
            >
              <span
                className="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
              >
                Comunicação e Oratória
              </span>
            </div>
          </div>
          <div
            aria-disabled={false}
            className="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
            onBlur={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            role="button"
            tabIndex={0}
          >
            <div
              className="MuiListItemText-root"
            >
              <span
                className="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
              >
                Cultura e Diversidade
              </span>
            </div>
          </div>
          <div
            aria-disabled={false}
            className="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
            onBlur={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            role="button"
            tabIndex={0}
          >
            <div
              className="MuiListItemText-root"
            >
              <span
                className="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
              >
                Liderança
              </span>
            </div>
          </div>
        </ul>
      </div>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-10 MuiGrid-grid-sm-4"
    >
      <div
        className="MuiPaper-root MuiCard-root MuiPaper-elevation1 MuiPaper-rounded"
      >
        <div
          className="MuiCardContent-root"
        >
          <h3
            className="MuiTypography-root MuiTypography-h6"
          >
            Acompanhamentos da Trilha
          </h3>
        </div>
        <ul
          className="MuiList-root MuiList-padding"
        >
          <div
            aria-disabled={false}
            className="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
            onBlur={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            role="button"
            tabIndex={0}
          >
            <div
              className="MuiListItemText-root"
            >
              <span
                className="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
              >
                Acompanhamento Psicológico
              </span>
            </div>
          </div>
          <div
            aria-disabled={false}
            className="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
            onBlur={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            role="button"
            tabIndex={0}
          >
            <div
              className="MuiListItemText-root"
            >
              <span
                className="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
              >
                Vivência de Autoconhecimento
              </span>
            </div>
          </div>
          <div
            aria-disabled={false}
            className="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
            onBlur={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            role="button"
            tabIndex={0}
          >
            <div
              className="MuiListItemText-root"
            >
              <span
                className="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
              >
                Palestras com Representatividades
              </span>
            </div>
          </div>
          <div
            aria-disabled={false}
            className="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
            onBlur={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            role="button"
            tabIndex={0}
          >
            <div
              className="MuiListItemText-root"
            >
              <span
                className="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
              >
                Mentorias com Especialistas
              </span>
            </div>
          </div>
          <div
            aria-disabled={false}
            className="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
            onBlur={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            role="button"
            tabIndex={0}
          >
            <div
              className="MuiListItemText-root"
            >
              <span
                className="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
              >
                Orientação Profissional
              </span>
            </div>
          </div>
          <div
            aria-disabled={false}
            className="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
            onBlur={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            role="button"
            tabIndex={0}
          >
            <div
              className="MuiListItemText-root"
            >
              <span
                className="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
              >
                Cultura e Diversidade
              </span>
            </div>
          </div>
        </ul>
      </div>
    </div>
  </div>
</main>
`;
